### DISCUSSION TITLE
Database grows a lot

### DISCUSSION THREAD

**Discussion by ivan11paths** [photoprism/photoprism#4210] [General]
I wanted to make a maintenance consultation. 20 days or so ago I had quite a few problems with face detection and there were a lot of conflicts. Warnings of this type appeared in the logs:

```
time="2024-04-23T07:06:05Z" level=warning msg="faces: LDTRRS73A5FHTVSO3BFDING7DRNCGLCG has ambiguous subject jrjf5c21a9t1y4mq"
time="2024-04-23T07:06:13Z" level=warning msg="faces: failed removing merged clusters for subject jrjf5c21a9t1y4mq"
time="2024-04-23T07:06:13Z" level=warning msg="faces: GJREMEANYQBZTXFZQFV4JF57ADL2XDDG has ambiguous subject jr4kq31drbrykmg2"
time="2024-04-23T07:06:17Z" level=warning msg="faces: failed removing merged clusters for subject jr4kq31drbrykmg2"
```
I think this may be related to this issues https://github.com/photoprism/photoprism/issues/3124

Today I realized that my database has grown at an enormous rate since that date. Above all, the bulk is in the errors (1657Mb) and markers (852Mb) tables. I would like to know if I can clean those tables somehow. I understand that the error one does, but the makers one, can I do something?

All the best,
Ivan.

**Comment by ivan11paths** [comment]
No comments here? This makes me sad.

**Reply by lastzero** [reply]
You can simply click the trash icon under Library > Errors. As for the markers, these are needed for facial recognition.
